test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,30 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Server;
+import com.googlecode.jmxtrans.model.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
I see, no problem. So I'll keep the code in my fork (because I need the agent with CloudWatch support).